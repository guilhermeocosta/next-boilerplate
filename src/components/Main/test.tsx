import { render, screen } from '@testing-library/react';

import Main from '.';

describe('<Main />', () => {
  it('should render a heading element', () => {
    const { container } = render(<Main />);

    expect(
      screen.getByRole('heading', { name: /React avan√ßado/i }),
    ).toBeInTheDocument();

    expect(container).toMatchSnapshot();
  });
});
