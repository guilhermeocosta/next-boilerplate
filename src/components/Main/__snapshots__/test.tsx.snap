// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Main /> should render a heading element 1`] = `
<div>
  <main>
    <h1>
      React avan√ßado
    </h1>
  </main>
</div>
`;
